### PR TITLE
Added an permanent discord invite link

### PR DESCRIPTION
**Pull request by rahul-mishra-oops**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 ## About EddieBotUI
 
-EddieBotUI is a web application built to display all kinds of information about the **Eddie Jaoude Discord community** - [click to join](https://discord.gg/fPss8W9) - like users biography, usage stats, badges and more.
+EddieBotUI is a web application built to display all kinds of information about the **Eddie Jaoude Discord community** - [click to join](https://discord.gg/dCUbX7tYUf) - like users biography, usage stats, badges and more.
```
Comment (BOLT04) [inline README.md:33]: This should be updated to be consistent with other repos, how about this:
```suggestion
EddieBotUI is a web application built to display all kinds of information about the **EddieHub Discord community** - [click to join](http://discord.eddiehub.org) - like users biography, usage stats, badges and more.
```

**Review by BOLT04 — COMMENTED** [review]
Hi @rahul-mishra-oops, thank you for your contribution 👍. Just left a comment to make the link and community name consistent with other repos 🙂

**Review by suitupstan — CHANGES_REQUESTED** [review]
Please see the comments left above.

**Review by suitupstan — CHANGES_REQUESTED** [review]
Duplicate of #49

**Comment by eddiejaoude** [comment]
Thank you for the contribution. I am going to close it has been fixed

**Pull request closed without merging**